README.md

```diff
@@ -2,5 +2,48 @@
 
 **Measurement framework for performance metrics.**
 
-This repo is being forked from [validate_drp](https://github.com/lsst/validate_drp).
-Watch the [DM-7692](https://jira.lsstcorp.org/browse/DM-7692) story for plans and progress.
+Packages can use `lsst.validate.base` to report metric measurements to LSST Data Management's [SQUASH](https://squash.lsst.codes) dashboard.
+This package is part of the [LSST Science Pipelines](https://pipelines.lsst.io).
+
+`lsst.validate.base` provides:
+
+- A schema for defining performance **metrics** (things that *can* be measured) and **specifications** (threshold values of a metric). The `lsst.validate.base.Metric` and `lsst.validate.base.Specification` classes provide Python access to metrics and specifications.
+- An `lsst.validate.base.MeasurementBase` base class that can be inherited to make **measurement** classes. Measurement classes create well-structured self-describing datasets that track units, input datasets, and even measurement code parameters.
+- An `lsst.validate.base.BlobBase` base class that can be inherited to make **blobs**: containers for semantically rich data.
+- An `lsst.validate.base.Job` class that collects multiple measurements, their metrics, and blob datasets, and provides a JSON file that can be submitted to [SQUASH](https://squash.lsst.codes).
+
+## Installation
+
+This package is part of the [LSST Science Pipelines](https://pipelines.lsst.io).
+You can learn how to install the Pipelines at https://pipelines.lsst.io/install.
+
+Then, to install this `validate_base` package from source:
+
+```
+git clone https://github.com/lsst/validate_base
+cd validate_base
+eups declare -r . -t $USER validate_base git
+setup -r . -t $USER
+scons
+```
+
```

Comment:
Sometimes people are interested in knowing how to short circuit the install. You see people say "use pip install" in READMEs. You are covering the "I want to make changes, how do I build this?" information.